lexicon/providers/plesk.py

```diff
@@ -0,0 +1,263 @@
+import logging
+
+import requests
+import xmltodict
```

Comment:
add try catch around xmltodict import

similar to: https://github.com/AnalogJ/lexicon/blob/master/lexicon/providers/route53.py#L9-L13 and other optional deps.